emsdk/patches/fix_emulate_pointers_dlsym.patch

```diff
@@ -0,0 +1,62 @@
+diff --git a/src/passes/FuncCastEmulation.cpp b/src/passes/FuncCastEmulation.cpp
+index 9d232c9..35d6d68 100644
+--- a/emsdk/binaryen/src/passes/FuncCastEmulation.cpp
++++ b/emsdk/binaryen/src/passes/FuncCastEmulation.cpp
+@@ -167,13 +167,14 @@ private:
+ struct FuncCastEmulation : public Pass {
+   void run(PassRunner* runner, Module* module) override {
+     Index numParams =
+-      std::stoul(runner->options.getArgumentOrDefault("max-func-params", "16"));
++      std::stoul(runner->options.getArgumentOrDefault("max-func-params", "61"));
```

Comment:
It should be possible to pass this as a param, since https://github.com/WebAssembly/binaryen/pull/3411 included in version_99